### PR TITLE
Added current_version to patch database script.

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

CI and WL environments are gonna start using a latest ddl image for patching the database. Instead of updating the manifest every time theres a release, we can just update the current_version variable in the python script. This means we don't have to have a config variable of the ddl_version. 

# What has changed

- Added current_version variable to the patch_database script.

# Checklist
Reminder: Make sure to version tag this after the PR is merged
* [ ] Updated patch number
* [ ] Updated version_tag

# How to test?

- Run the patch database script in your GCP environment and it should still work

# Links

[Trello](https://trello.com/c/UWyM9NQs)

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[Fix acceptance test data build up issue (13)](https://trello.com/c/UWyM9NQs/667-fix-acceptance-test-data-build-up-issue-13)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,7 +9,8 @@
 # Checklist
 Reminder: Make sure to version tag this after the PR is merged
 * [ ] Updated patch number
-* [ ] Updated version_tag
+* [ ] Updated version_tag in `ddl_version.sql`
+* [ ] Updated current_version in `patch_database.py`
```
Comment (AdamHawtin) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: 👍 Good reminder to any future changers or reviewers

File: patch_database.py [hunk 2]
```diff
@@ -7,6 +7,8 @@
 
 PATCHES_DIRECTORY = Path(__file__).parent.joinpath('patches')
 
+current_version = 'v2.2.0'
```
Comment (AdamHawtin) [inline patch_database.py:10]: Worth a comment on this line to say this should match the sql script?

File: patch_database.py [hunk 3]
```diff
@@ -65,11 +67,11 @@ def main():
                           f"'{Config.DB_USESSL}") as db_connection:
         db_connection.set_session(autocommit=False)
         with db_connection.cursor() as db_cursor:
-            if Config.DDL_VERSION_TAG == get_current_database_version_tag(db_cursor):
```
Comment (liamtoozer) [inline patch_database.py:68]: Could remove `DDL_VERSION_TAG` from the config now too if we're hardcoding it?

File: patch_database.py (hunk 2 above)
Comment (ryangrundy7) [reply patch_database.py:11]: Done 👍

File: patch_database.py (hunk 3 above)
Comment (ryangrundy7) [reply patch_database.py:68]: Removed it 👍

**Pull request merged**